src/modules/Storage.rb

```diff
@@ -367,13 +368,18 @@ def FinishLibstorage
     end
 
 
+    def DefaultSubvolumeName()
```

Comment:
I though that the agreement was using Ruby-style names for the new methods, like ```default_subvolume_name```